### PR TITLE
Share panel extra buttons

### PR DESCRIPTION
**Pull request by katerynabugaieva**
after final updates in design, share panel needs:

 [ + ]  linkedin button and its sharing

 [ + ]  update for all button icons

 [ + ]  xing button and its sharing logic
this PR about it.

Also, it will affect #40 Meta tags

### CODE REVIEW

File: src/components/share-panel/share-panel.tsx [hunk 1]
```diff
@@ -63,16 +72,19 @@ export const SharePanel: React.FC = () => {
...
             <IconTwitter color="#668cff" />
           </TwitterShareButton>
         </SocialLinkItem>
+        <SocialLinkItem>
+          <LinkedinShareButton title={title} url={shareUrl}>
+            <IconLinkedIn color="#668cff" />
+          </LinkedinShareButton>
+        </SocialLinkItem>
+        <SocialLinkItem>
+          <IconXing color="#668cff" />
```
Comment (feedm3) [inline src/components/share-panel/share-panel.tsx]: Nothing happens if I click on it 🧐

File: src/components/share-panel/share-panel.tsx [hunk 2]
```diff
@@ -63,16 +72,19 @@ export const SharePanel: React.FC = () => {
             <IconEmail />
           </EmailShareButton>
```
Comment (feedm3) [inline src/components/share-panel/share-panel.tsx:76]: Can you remove the `body` please?

**Review by feedm3 — CHANGES_REQUESTED** [review]

File: src/components/share-panel/share-panel.tsx (hunk 1 above)
Comment (katerynabugaieva) [reply src/components/share-panel/share-panel.tsx]: there is no implementation yet.
Perhaps, have you any suggestion for XING-sharing?

Comment (feedm3) [reply src/components/share-panel/share-panel.tsx]:
```
https://www.xing.com/social/share/spi?url={URL}
```

Example: https://www.xing.com/social/share/spi?url=https%3A%2F%2Fsatellytes.com

File: src/components/share-panel/share-panel.tsx [hunk 3]
```diff
@@ -83,7 +86,13 @@ export const SharePanel: React.FC = () => {
           </LinkedinShareButton>
         </SocialLinkItem>
         <SocialLinkItem>
-          <IconXing color="#668cff" />
+          <XingShareButton
+            href={'https://www.xing.com/spi/shares/new?url=' + shareUrl}
+            title={title}
+            target="_blanc"
```
Comment (feedm3) [inline src/components/share-panel/share-panel.tsx]: `_blank`

**Review by feedm3 — CHANGES_REQUESTED** [review]

**Review by feedm3 — APPROVED** [review]

**Pull request merged**